.github/bin/install-kythe.sh

```diff
@@ -42,5 +42,9 @@ else
     wget --no-verbose -O kythe.tar.gz \
       "https://github.com/kythe/kythe/releases/download/$KYTHE_VERSION/kythe-$KYTHE_VERSION.tar.gz"
     tar -xzf kythe.tar.gz
+    # Bazel renamed @com_google_protobuf to @protobuf while Kythe is still
+    # using it in their BUILD targets. Manually rename until the fixed Kythe is
+    # released.
+    sed -i 's/@com_google_protobuf/@protobuf/' "kythe-${KYTHE_VERSION}/BUILD"
```

Comment:
there is also repo-mapping, maybe it can work with that ? https://bazel.build/rules/lib/repo/http#http_archive-repo_mapping

Reply:
I thought of that. We used it in past. But verible is now using the Bazel modules & I don't see a way how to do remapping with the modules.